styles/find-and-replace.less

```diff
@@ -166,6 +166,25 @@ atom-workspace.find-visible {
       margin-right: @component-padding;
     }
   }
+
+  .wrap-icon {
+    @wrap-size: 96px;
+
+    display: none;
+    position: absolute;
+    left: 50%;
+    top: @wrap-size * -1.5;
+    margin-left: @wrap-size * -0.5;
+    background: rgba(136,136,136, 0.2);
+    border-radius: 10px;
```

Comment:
Done https://github.com/atom/find-and-replace/commit/386cbcb7ccf93fae9d4d25513d5f368baa00a211#diff-0c91d4121c1775ed1f0c74e547086288R179